features/admin.applications.v1/pages/applications-settings.tsx

```diff
@@ -0,0 +1,430 @@
+ */
+interface ApplicationsSettingsPropsInterface extends IdentifiableComponentInterface {
+    /**
+     * Is the mandateSSA enabled.
+     */
+    mandateSSA?: boolean;
+    /**
+     * Is the requireAuthentication enabled.
+     */
+    authenticationRequired?: boolean;
+    /**
+     * JWKS Endpoint Url.
+     */
+    ssaJwks?: string;
+    /**
+     * DCR Endpoint Url.
+     */
+    dcrEndpoint?: string;
+    /**
+     * Is fapi complience enforced for the dcr apps.
+     */
+    enableFapiEnforcement?:boolean
+}
+
+/**
+ * Form values interface.
+ */
+export interface ApplicationsSettingsFormValuesInterface {
+    /**
+     * Is the mandateSSA enabled.
+     */
+    mandateSSA?: boolean;
+    /**
+     * Is the requireAuthentication enabled.
+     */
+    authenticationRequired?: boolean;
+    /**
+     * JWKS Endpoint Url.
+     */
+    ssaJwks?: string;
+    /**
+     * Is fapi complience enforced for the dcr apps.
+     */
+    enableFapiEnforcement?:boolean
+}
+
+/**
+ * Proptypes for the applications settings form error messages.
+ */
+export interface ApplicationsSettingsFormErrorValidationsInterface {
+    /**
+     *  Error message for the JWKS URL.
+     */
+    ssaJwks?: string;
+}
+
+const FORM_ID: string = "applications-settings";
+
+
+/**
+ * Form to edit applications settings.
+ *
+ * @param props - Props injected to the component.
+ * @returns Functional Component.
+ */
+export const ApplicationsSettingsForm: FunctionComponent<ApplicationsSettingsPropsInterface> = (
+    props: ApplicationsSettingsPropsInterface
+): ReactElement => {
+    
+    const { 
+        mandateSSA,
+        authenticationRequired,
+        ssaJwks,
+        dcrEndpoint,
+        enableFapiEnforcement,
+        ["data-componentid"]: componentId 
+    } = props;
+   
+    const dispatch: Dispatch = useDispatch();
+    const { t } = useTranslation();
+    const { getLink } = useDocumentation();
+
+    const [ isAuthenticationRequired = true , setAuthenticationRequired ] = useState<boolean>(authenticationRequired);
+    const [ isMandateSSA = !isAuthenticationRequired, setMandateSSA ] = useState<boolean>(mandateSSA);
+    const [ isEnableFapiEnforcement, setEnableFapiEnforcement ] = useState<boolean>(enableFapiEnforcement);
+    const [ ssaJwksState, setSsaJwks ] = useState<string>(ssaJwks);
+    const [ dcrEndpointState, setDCREndpoint ] = useState<string>(dcrEndpoint);
+    const [ isSubmitting, setIsSubmitting ] = useState<boolean>(false);
+
+
+    const {
+        data: dcrConfigs,
+    } = useGetDCRConfigurations();
+
+    // set initial values
+    useEffect(() => {
+        if (dcrConfigs) {
+            setMandateSSA(dcrConfigs?.mandateSSA);
+            setAuthenticationRequired(dcrConfigs?.authenticationRequired);
+            setEnableFapiEnforcement(dcrConfigs?.enableFapiEnforcement);
+            setSsaJwks(dcrConfigs?.ssaJwks);
+        }
+    }, [ dcrConfigs ]);
+
+    useEffect(() => {
```

Comment:
It's better to add doc comments for useEffects as well, explaining the functionality. Check other places as well.

Reply:
Added with [d54ec35](https://github.com/wso2/identity-apps/pull/5964/commits/d54ec35c7ee212f1039c9afea0052b20884b03c3)